7-bank-project/README.md

```diff
@@ -16,6 +16,6 @@ In this project, you'll learn how to build a fictional bank. These lessons inclu
 
 These lessons were written with :hearts: by [Yohan Lasorsa](https://twitter.com/sinedied).
 
-If you're interested to learn how to build the [server API](./api/README) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
+If you're interested to learn how to build the [server API](https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/7-bank-project/api/README.md) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
```

Comment:
I think this can be a relative link

Reply:
I agree. Thank you for your feedback. I've changed it.

Reply:
@jlooper  Do I click the "Resolve conversation"?  I'm new, so I don't know the common protocol. :(

Reply:
This looks good, I'll merge! thank you